MekHQ/src/mekhq/campaign/universe/StarUtil.java

```diff
@@ -624,7 +624,8 @@ public static String getHPGClass(Integer hpg) {
 
     public static String getIconImage(Planet planet) {
         if (!planetIconDataLoaded) {
-            try (BufferedReader reader = new BufferedReader(new FileReader(new File("data", PLANET_ICON_DATA_FILE)))) {
+            try (FileReader fr = new FileReader(new File("data", PLANET_ICON_DATA_FILE));
+                 BufferedReader reader = new BufferedReader(fr)) {
```

Comment:
How is this functionally different from the previous syntax?

Reply:
Try with resources doesn't close inner declarations, so the new functionality actually closes the FileReader instead of letting it leak.